### PR TITLE
Use google tag manager plugin

### PR DESCRIPTION
**Pull request by gris**
## What's the purpose of this pull request?
<em>Considering the context, what is the problem we'll solve? Where in VTEX's big picture our issue fits in? Write a tweet about the context and the problem itself.</em>
To use the new google tag manager plugin

## How it works? 
<em>Tell us the role of the new feature, or component, in its context.</em>

## How to test it?
<em>Describe the steps with bullet points. Is there any external reference, link, or example?</em> 
See that the script is being loaded into the webpage.

## References
<em>Spread the knowledge: is this any content you used to create this PR that is worth sharing?</em>  

<em>Extra tip: add references to related issues or mention people important to this PR may be good for the documentation and reviewing process</em>

### CODE REVIEW

File: gatsby-browser.js [hunk 1]
```diff
@@ -1,6 +1 @@
-import React from 'react'
-import GTMProvider from '@vtex/gatsby-theme-store/src/sdk/pixel/GTM/index'
-
-export const wrapRootElement = ({ element }) => (
-  <GTMProvider gtmId="GTM-TT2MDM3">{element}</GTMProvider>
-)
+// noop
```
Comment (klzns) [inline gatsby-browser.js:1]: Maybe remove this file?

**Review by klzns — APPROVED** [review]

File: gatsby-browser.js (hunk 1 above)
Comment (gris) [reply gatsby-browser.js]: Done!

**Comment by tlgimenes** [comment]
Please remove this repo's gatsby-browser.js

**Review by tlgimenes — CHANGES_REQUESTED** [review]
Please remove this repo's gatsby-browser.js

**Comment by gris** [comment]
It was already removed @tlgimenes

**Review by tlgimenes — APPROVED** [review]

**Pull request merged**